bin/compile

```diff
@@ -43,19 +50,15 @@ if [ ! -f $DB ]; then
   if [ ! -f $ARCHIVE ]; then
     echo "-----> $MAXMIND_FULL_DB_NAME: Database does not exist in cache, downloading."
 
-    url="http://geolite.maxmind.com/download/geoip/database/$MAXMIND_FULL_DB_NAME"
+    url="https://download.maxmind.com/app/geoip_download?edition_id=${MAXMIND_FULL_DB_NAME}&license_key=${MAXMIND_KEY}&suffix=tar.gz"
```

Comment:
Do we know whether MaxMind has a limitation on this download? I can't seem to find a clear word on this in their documentation. Not that the 10th deploy on a given day doesn't go through :)